components/avatar/avatar.css

```diff
@@ -0,0 +1,33 @@
+@import url("../style.css");
+
+.avatar {
+  display: inline-block;
+  border-radius: 50%;
```

Comment:
border radius is repeating multiple time, you add general styles in one class and add that class to every avatar.

Reply:
I thought it's a single line of code and by default avatars are circular, so I used that in all classes.